### DISCUSSION TITLE
Ability to edit attributes in a more efficient way

### DISCUSSION THREAD

**Discussion by Jdgilby** [Riverscapes/QRiS#462] [Ideas & Feature Requests]
I understand this is complex and has been difficult since we switched to storing data within 3 feature classes. Right now attributes are stored as JSON in one metadata attribute. This makes editing features attributes quite restrictive. Right now for existing features you have to go to the form view of the attribute table and edit one feature at a time. 

Is there any possibility that under the current structure you can make changes to multiple features at once?
For example:
![image](https://github.com/user-attachments/assets/3b0dcca4-acfa-42e6-99bf-65f15d2b98c7)
Using the editor here you can apply changes to all selected features. 

Just something to think about since making changes to a lot of features now requires the following workflow. Extract selected features to temporary layer, make the necessary changes, delete features from project, reimport temporary layer with changes. 

@joewheaton @philipbaileynar @KellyMWhitehead

**Comment by ecologis** [comment]
As a user (not developer) I've been doing that workflow too, and I hope there is ultimately a backend workaround to making feature attributes editable.
